src/Argu/project.json

```diff
@@ -0,0 +1,65 @@
+{
+    "version": "3.0.0-beta02",
+    "buildOptions": {
+        "debugType": "portable",
+        "compilerName": "fsc",
+        "define": [
+            "CORE_CLR"
+        ],
+        "compile": {
+            "includeFiles": [
+                "AssemblyInfo.fs",
+                "Types.fs",
+                "Attributes.fs",
+                "Utils.fs",
+                "ConfigReaders.fs",
+                "UnionArgInfo.fs",
+                "PreCompute.fs",
+                "UnParsers.fs",
+                "ParseResults.fs",
+                "Parsers/Common.fs",
+                "Parsers/Cli.fs",
+                "Parsers/KeyValue.fs",
+                "ArgumentParser.fs"
+            ]
+        }
+    },
+    "tools": {
+        "dotnet-compile-fsc": {
+        "version": "1.0.0-preview2-*",
+        "imports": "dnxcore50"
+        }
+    },
+    "frameworks": {
+        "netstandard1.6": {
+            "buildOptions": {
+                "define": [
+                    "NETSTANDARD1_5",
+                    "NETSTANDARD1_6"
+                ]
+            },
+            "dependencies": {
+                "System.Xml.XDocument": "4.0.11",
+                "Microsoft.FSharp.Core.netcore": "1.0.0-alpha-160629",
```

Comment:
Can this be removed and https://www.nuget.org/packages/FSharp.Core/4.0.1.7-alpha used instead below instead of `4.0.0.1`?